src/qt/rpcconsole.cpp

```diff
@@ -975,6 +975,20 @@ void RPCConsole::updateNetworkState()
     }
 
     ui->numberOfConnections->setText(connections);
+
+    QString local_addresses;
+    std::map<CNetAddr, LocalServiceInfo> mapHosts = clientModel->getLocalAddresses();
+    for (auto it = mapHosts.begin(); it != mapHosts.end(); it++) {
+        if (it != mapHosts.begin()) local_addresses += ", ";
+        if (it->first.IsI2P())
+            local_addresses += QString::fromStdString(it->first.ToStringAddr());
+        else
+            local_addresses += QString::fromStdString(it->first.ToStringAddr()) + ":" + QString::number(it->second.nPort);
+    }
```

Comment:
Maybe?
```c++
QString local_addresses;
std::map<CNetAddr, LocalServiceInfo> hosts = clientModel->getLocalAddresses();
for (const auto& [addr, info] : hosts) {
    local_addresses += QString::fromStdString(it->first.ToStringAddr());
    if (!addr.IsPrivacyNet()) local_addresses += ":" + QString::number(it->second.nPort);
    local_addresses += ", ";
}
local_addresses.chop(2); // remove last ", "
```

Reply:
This is more beautiful and understandable. Replaced the `it->first` & `it->second` with `addr` & `info`:

```
    for (const auto& [addr, info] : hosts) {
        local_addresses += QString::fromStdString(addr.ToStringAddr());
        if (!addr.IsPrivacyNet()) local_addresses += ":" + QString::number(info.nPort);
        local_addresses += ", ";
    }
    local_addresses.chop(2); // remove last ", "
```